.circleci/config.yml

```diff
@@ -64,9 +64,19 @@ commands:
         type: string
     steps:
       - run: |
-          amazon-linux-extras install python3.9 java-openjdk11 -y
-          yum install -y git tar gcc gcc-c++ file lsof which procps
-          ln -s /usr/bin/python3.9 /usr/bin/python3
+          amazon-linux-extras install java-openjdk11 -y
+          yum install wget make gcc gcc-c++ openssl-devel bzip2-devel libffi-devel zlib-devel file lsof which procps tar git -y
+          
+          cd /tmp
+          wget https://www.python.org/ftp/python/3.9.6/Python-3.9.6.tgz
+          tar -xvf Python-3.9.6.tgz
+          cd Python-3.9.6
+          ./configure --enable-optimizations --enable-shared
```

Comment:
We install a single python version for both deployment and testing steps, so:
* `shared` is needed for the dynamic library usage on the testing stage
* `ldconfig` sets the path to the dynamic library globally for all the sessions (so it could be found for both steps)